src/bin/pgcopydb/ld_wal2json.c

```diff
@@ -120,18 +125,13 @@ parseWal2jsonMessage(StreamContext *privateContext,
 	/* most actions share a need for "schema" and "table" properties */
 	JSON_Object *jsobj = json_value_get_object(json);
 
-	char *schema = NULL;
-	char *table = NULL;
-
-	schema = (char *) json_object_dotget_string(jsobj, "message.schema");
-	table = (char *) json_object_dotget_string(jsobj, "message.table");
+	LogicalMessageRelation table = { 0 };
 
-	if (schema == NULL || table == NULL)
+	if (!SetMessageRelation(jsobj, &table, privateContext->transformPGSQL))
```

Comment:
@dimitri @shubhamdhama Since we do this for every DML operation, I'm worried about the additional network latency introduced by this. Can we just copy the function from Postgres source which performs escaping?

Reply:
Ahh, never mind. It doesn't do any networking.! Sorry for the false alarm.